### DISCUSSION TITLE
Why not enable `strict` in the base config?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [NiGhTTraX/ts-monorepo#317] [Q&A] [answered]
This is a popular repo, used by many TypeScript newbies (like me) for learning purposes - and also for copy-pasting purposes :D

Why not promote the use of [`strict`](https://www.typescriptlang.org/tsconfig#strict) in the base [`tsconfig.build.json`](https://github.com/NiGhTTraX/ts-monorepo/blob/82b440c77f1ec20b234d69eaecc7757b3bb957d0/tsconfig.build.json)?

**Comment by NiGhTTraX** [comment, marked as answer]
The repo started out with minimal configs to let people add what they need on top, but I think it's fair to add `strict: true`. Done in https://github.com/NiGhTTraX/ts-monorepo/commit/3fb637e971284abf3aa1fc42f39f9cb5d0f52d25.
